storage/container_test.go

```diff
@@ -181,12 +181,12 @@ func (s *ContainerSuite) TestCreateContainerIfNotExists(c *chk.C) {
 
 func (s *ContainerSuite) TestCreateContainerIfExists(c *chk.C) {
 	cli := getBlobClient(c)
+	rec := cli.client.appendRecorder(c)
+	defer rec.Stop()
+
 	cnt := cli.GetContainerReference(containerName(c))
 	cnt.Create(nil)
 	defer cnt.Delete(nil)
-	rec := cli.client.appendRecorder(c)
-	cnt.bsc = &cli
```

Comment:
You're not assigning this line anymore....not sure if you need it?

Reply:
Correct, it's not used so I removed it.